src/Components/SmartComponents/Reports/DownloadCVEsReport.js

```diff
@@ -50,7 +51,7 @@ const DownloadCVEsReport = ({
             ({ data: cves, meta } = await getCveListByAccount({
                 ...params,
                 ...DownloadReport.defaultParams,
-                sort: params.sort || '-cvss_score',
+                sort: sortParam,
```

Comment:
Careful here, `DownloadCVEsReport.js` is responsible for generating both CVE report (exported from CVE list page) and custom CVE report (exported from Reports page). If you export from CVE list page which is sorted the sorting inside the PDF will be retained. By removing `params.sort` you will break this. So I believe something like this `params.sort || sortParam` would work, but double check

Reply:
returned to `params.sort || '-cvss_score',`